helpers/userHelper.js

```diff
@@ -10,5 +14,17 @@ export const getInitials = (user) => {
   return `${firstName.charAt(0)}${lastName.charAt(0)}`;
 };
 
-export const passwordRegex =
-  /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d\W_]{8,}$/i;
+export const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d\W_]{8,}$/i;
+
+export async function updateUser(updateFields) {
+  try {
+    const api = gpApi.user.updateUser;
+
+    const response = await gpFetch(api, updateFields);
+    const { user } = response;
+    setUserCookie(user);
```

Comment:
This will only work on client side... do we nee do add 'use client' here too? are we using passwordRegex on any server component?

Reply:
If this code gets run server-side, it's just a no-op.. so I think we're good w/o adding `'use client'` here.. 

All of these are used in `use client` files.. 
<img width="671" alt="image" src="https://github.com/thegoodparty/gp-webapp/assets/496983/8628199d-d1ef-4d44-ac38-91328215e34e">

But, like I said, even if it were accidentally added in a server page, it'd just be a no-op, so no real need to add something that won't have any real functional impact, IMHO..